packages/zapp/console/src/components/Executions/Tables/NodeExecutionsTable.tsx

```diff
@@ -26,16 +35,57 @@ const scrollbarPadding = scrollbarSize();
  * NodeExecutions are expandable and will potentially render a list of child
  * TaskExecutions
  */
-export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({
-  abortMetadata,
-  nodeExecutions,
-}) => {
+export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({ abortMetadata }) => {
   const [selectedExecution, setSelectedExecution] = React.useState<NodeExecutionIdentifier | null>(
     null,
   );
+  const [nodeExecutions, setNodeExecutions] = React.useState<NodeExecution[]>([]);
```

Comment:
nit:
update all standard react hooks to be imported from 'react' file-wide, so we can use them as 
```
  const [nodeExecutions, setNodeExecutions] = useState<NodeExecution[]>([]);
```
which is much shorter, and benefits in readability

Reply:
done